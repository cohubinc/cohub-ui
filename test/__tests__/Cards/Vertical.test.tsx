import React from "react";
import { mount } from "enzyme";

import { Card } from "dist";

describe("Vertical", () => {
  it("renders without crashing", () => {
    const wrapper = mount(<Card.Vertical title="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});
