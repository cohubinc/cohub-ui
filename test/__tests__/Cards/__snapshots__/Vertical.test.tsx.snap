// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vertical renders without crashing 1`] = `
<Vertical
  title="test"
>
  <styled.div
    style={
      Object {
        "boxShadow": "0px -1px 1px rgba(0, 0, 0, 0.03), 0px 1px 1px rgba(0, 0, 0, 0.14)",
        "width": "240px",
      }
    }
  >
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bxivhb",
            "isStatic": false,
            "lastClassName": "gRqUsw",
            "rules": Array [
              "
  display: flex;
  flex-direction: column;
  background-color: ",
              "hsl(0, 0%, 100%)",
              ";
  border-radius: var(--default-border-radius);
  padding-bottom: 1rem;

  img {
    border-top-left-radius: var(--default-border-radius);
    border-top-right-radius: var(--default-border-radius);
  }
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bxivhb",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      style={
        Object {
          "boxShadow": "0px -1px 1px rgba(0, 0, 0, 0.03), 0px 1px 1px rgba(0, 0, 0, 0.14)",
          "width": "240px",
        }
      }
    >
      <div
        className="sc-bxivhb gRqUsw"
        style={
          Object {
            "boxShadow": "0px -1px 1px rgba(0, 0, 0, 0.03), 0px 1px 1px rgba(0, 0, 0, 0.14)",
            "width": "240px",
          }
        }
      >
        <div
          className="m-auto block"
        />
        <div
          className="mx-1 mt-05"
        >
          <Component
            block={true}
            className=""
          >
            <span
              className=""
              data-qa="text"
              style={
                Object {
                  "color": "hsl(0, 2%, 22%)",
                  "display": "block",
                  "fontFamily": "Inter",
                  "fontSize": 16,
                  "fontWeight": 400,
                  "textAlign": "initial",
                  "textTransform": "initial",
                }
              }
            >
              test
            </span>
          </Component>
          <Component
            block={true}
            className=" mt-025"
          />
        </div>
      </div>
    </StyledComponent>
  </styled.div>
</Vertical>
`;
